<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCourse</class>
 <widget class="QWidget" name="AddCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>148</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>307</width>
    <height>148</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>405</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Course</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 12, 10);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="DataInput">
     <item row="0" column="0">
      <widget class="QLabel" name="label_CourseName">
       <property name="styleSheet">
        <string notr="true">font: 700 11pt &quot;Graduate&quot; ;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Course Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_CourseName">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(192, 191, 188);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_ECTSPoints">
       <property name="styleSheet">
        <string notr="true">font: 700 11pt &quot;Graduate&quot; ;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>ECTS Points</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_ECTSPoints">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 245, 244);</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_ExamType">
       <property name="styleSheet">
        <string notr="true">font: 700 11pt &quot;Graduate&quot; ;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Exam Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_ExamType">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(192, 191, 188);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
